src/scirpy/pl/_logoplots.py

```diff
@@ -0,0 +1,177 @@
+from collections.abc import Sequence
+from typing import Literal, Union
+
+import palmotif as palm
+import pandas as pd
+from IPython.display import SVG
+
+from scirpy.get import airr as get_airr
+from scirpy.util import DataHandler
+
+
+@DataHandler.inject_param_docs()
+def logoplot_cdr3_motif(
+    adata: DataHandler.TYPE,
+    chains: Union[
+        Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"],
+        Sequence[Literal["VJ_1", "VDJ_1", "VJ_2", "VDJ_2"]],
+    ] = "VDJ_1",
+    airr_mod="airr",
+    airr_key="airr",
+    chain_idx_key="chain_indices",
+    cdr3_col: str = "junction_aa",
+    *,
+    by: Sequence[Literal["gene_segment", "clonotype", "length"]] = "length",
+    target_col: Union[None, str] = None,
+    gene_annotation: Union[None, list] = None,
+    clonotype_id: Union[None, list] = None,
+    clonotype_key: Union[None, str] = None,
+    cdr_len: int,
+    plot: bool = True,
```

Comment:
I think we don't need that option, better to always use matplotlib. If the user wants to save a logo, they can do that through matplotlib.